po/nl.po

```diff
@@ -7,504 +7,496 @@ msgid ""
+
 #: data/org.freedesktop.Piper.appdata.xml.in.in:10
 msgid ""
 "Piper is a graphical user interface to configure gaming mice. Configuration "
 "options include changing the resolution (DPI) of the mouse, adding and "
 "removing profiles, setting LED colors and changing button behaviors."
 msgstr ""
-"Piper is een grafische toepassing voor het instellen van gamemuizen. Zo kun "
-"je onder meer de resolutie (DPI) van je muis instellen, profielen toevoegen "
-"en verwijderen, en LED-kleuren en knoptoewijzingen aanpassen."
+"Piper is een grafische toepassing voor het instellen van gamingmuizen. Zo "
+"kun je onder meer de resolutie (dpi) van je muis instellen, profielen "
+"toevoegen en verwijderen en ledkleuren en knoptoewijzingen aanpassen."
 
 #: data/org.freedesktop.Piper.appdata.xml.in.in:16
 msgid ""
 "Piper requires libratbag’s ratbagd, the daemon to actually communicate with "
 "the mice. Piper is merely a front end to ratbagd, ratbagd must be installed "
 "and running when Piper is launched."
 msgstr ""
-"Piper is afhankelijk van libratbag's ratbagd, de achtergronddienst die "
+"Piper is afhankelijk van libratbags ratbagd, de achtergronddienst die "
 "daadwerkelijk met de muis communiceert. Piper is slechts een grafische schil "
-"hieromheen. ratbagd moet dus geïnstalleerd en actief zijn voordat je Piper "
-"opstart."
+"hieromheen. ratbagd dient dus geïnstalleerd en actief te zijn voordat je "
+"Piper opstart."
 
-#: data/org.freedesktop.Piper.appdata.xml.in.in:33
+#: data/org.freedesktop.Piper.appdata.xml.in.in:27
 msgid "The button configuraton page"
-msgstr "De knopinstellingenpagina"
+msgstr "De knopvoorkeurenpagina"
 
-#: data/org.freedesktop.Piper.appdata.xml.in.in:37
+#: data/org.freedesktop.Piper.appdata.xml.in.in:31
 msgid "The LED configuraton page"
-msgstr "De LED-instellingenpagina"
+msgstr "De ledvoorkeurenpagina"
 
-#: data/org.freedesktop.Piper.appdata.xml.in.in:41
+#: data/org.freedesktop.Piper.appdata.xml.in.in:35
 msgid "The resolution configuraton page"
-msgstr "De resolutie-instellingenpagina"
+msgstr "De resolutievoorkeurenpagina"
 
-#. Translators: section header for mapping one button's click to another.
-#: piper/buttondialog.py:141
+#: piper/buttondialog.py:156
 msgid "Button mapping"
-msgstr "Knopinstellingen"
+msgstr "Knopvoorkeuren"
 
-#. Translators: section header for assigning special functions to buttons.
-#: piper/buttondialog.py:150
+#: piper/buttondialog.py:172
 msgid "Special mapping"
-msgstr "Speciale instellingen"
+msgstr "Speciale voorkeuren"
+
+#: piper/buttondialog.py:182 data/ui/ResolutionRow.ui:95
+msgid "Disable"
+msgstr "Uitschakelen"
+
+#: piper/buttondialog.py:182
+msgid "Other"
+msgstr "Overig"
 
 #. Translators: the {} will be replaced with the button index, e.g.
 #. "Button 1 click".
-#: piper/buttondialog.py:222 piper/buttonspage.py:90
+#: piper/buttondialog.py:257 piper/buttonspage.py:105
 msgid "Button {} click"
-msgstr "Klik met de {}knop"
+msgstr "{}muisklik"
```

Comment:
Does this sound right? It will be like "1muisklik", "2muisklik" and so on.

Reply:
You're right: I may have interpreted that wrong. Will think of a better translation.